cli/daemon/run/tests.go

```diff
@@ -29,6 +31,15 @@ import (
 func (mgr *Manager) Check(ctx context.Context, appRoot, relwd string, codegenDebug bool) (buildDir string, err error) {
 	vcsRevision := vcs.GetRevision(appRoot)
 
+	exp, err := appfile.Experiments(appRoot)
+	if err != nil {
+		return "", err
+	}
+	expSet, err := experiments.NewSet(exp, nil)
```

Comment:
Seems like we're repeating these two everywhere. Would it make sense to move appfile.Experiments into experiments.NewSet instead?

Reply:
Not sure, it's not *everywhere*, so I don't think they should be unconditionally combined. Maybe makes sense to add a helper function `experiments.FromAppfile()` or something?

Reply:
figured since the first parameter of newSet is called fromAppFile it was only used in combination. But yeah, maybe helper would make sense

Reply:
Yeah, since I have a stacked PR on top I'll be lazy and defer changing this for now :)